hereditary/htaccess.txt

```diff
@@ -0,0 +1,54 @@
+Header set Access-Control-Allow-Origin *
+Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+Options +FollowSymLinks
+
+# Turn off MultiViews
+Options -MultiViews
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+
+# Rewrite engine setup
+RewriteEngine On
+RewriteBase /def
+
+# Rewrite rule to serve HTML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^ontology$ http://hereditary.dei.unipd.it/ontology/
+RewriteRule ^ontology/phenoclinical$ http://hereditary.dei.unipd.it/ontology/phenoclinical/
+RewriteRule ^ontology/genomics$ http://hereditary.dei.unipd.it/ontology/genomics/
+
+# Rewrite rule to serve RDF/XML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
+RewriteRule ^ontology/phenoclinical$ http://hereditary.dei.unipd.it/ontology/phenoclinical/schema/hero_phenoclinical.owl [R=303]
+RewriteRule ^ontology/genomics$ http://hereditary.dei.unipd.it/ontology/genomics/schema/hero_genomics.owl [R=303]
```

Comment:
the rewrite condition is per rule. You have to repeat the condition, otherwise this won't work as you intend